### PR TITLE
22w16b stuff

### PR DESCRIPTION
**Pull request by apple502j**
Mostly world loading changes (see #3107 for corresponding client-side changes)

### CODE REVIEW

File: mappings/net/minecraft/util/Util.mapping [hunk 1]
```diff
@@ -180,6 +180,31 @@ CLASS net/minecraft/class_156 net/minecraft/util/Util
 	METHOD method_43253 copyShuffled ([Ljava/lang/Object;Lnet/minecraft/class_5819;)Ljava/util/List;
 		ARG 0 array
 		ARG 1 random
+	METHOD method_43368 (Ljava/util/List;Ljava/lang/Void;)Ljava/util/List;
+		ARG 1 void_
```
Comment (Shnupbups) [inline mappings/net/minecraft/util/Util.mapping:184]: :concern:

**Review by Shnupbups — APPROVED** [review]

File: mappings/net/minecraft/util/Util.mapping (hunk 1 above)
Comment (apple502j) [reply mappings/net/minecraft/util/Util.mapping:184]: :tiny_completablefuture:

File: mappings/net/minecraft/world/level/storage/LevelSummary.mapping [hunk 2]
```diff
@@ -2,7 +2,7 @@ CLASS net/minecraft/class_34 net/minecraft/world/level/storage/LevelSummary
 	FIELD field_205 name Ljava/lang/String;
 	FIELD field_209 requiresConversion Z
 	FIELD field_23772 locked Z
-	FIELD field_23773 file Ljava/nio/file/Path;
+	FIELD field_23773 path Ljava/nio/file/Path;
```
Comment (enbrain) [inline mappings/net/minecraft/world/level/storage/LevelSummary.mapping]: I think `iconFile` best describes this field. Can be fixed later though.

**Review by enbrain — APPROVED** [review]

File: mappings/net/minecraft/world/level/storage/LevelSummary.mapping (hunk 2 above)
Comment (apple502j) [reply mappings/net/minecraft/world/level/storage/LevelSummary.mapping]: I'll fix that one now.

**Review by enbrain — APPROVED** [review]

**Pull request merged**